code/src/components/AddTask.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import Tasks from 'reducers/Task';
+
+const capitalize = (stringToCapitalized) => {
+  return stringToCapitalized.charAt(0).toUpperCase() + stringToCapitalized.slice(1);
+}
+const AddTask = () => {
+  const [inputValue, setInputValue] = useState('');
+
+  const dispatch = useDispatch()
+
+  const onFormSubmit = (event) => {
+    event.preventDefault();
+    const newTask = {
+      id: Date.now().toString(),
+      text: capitalize(inputValue),
```

Comment:
Is this a reference to some css-style? It looks so short but still does the work of making the first letter in the added task be big. My code looks like this: 

`text: inputValue.charAt(0).toUpperCase() + inputValue.slice(1)`

So much longer...